control-plane/agents/common/src/errors.rs

```diff
@@ -65,6 +65,14 @@ pub enum SvcError {
     VolumeNotFound { vol_id: String },
     #[snafu(display("Replica '{}' not found", replica_id))]
     ReplicaNotFound { replica_id: ReplicaId },
+    #[snafu(display("{} '{}' is already shared by {}", kind.to_string(), id, share))]
```

Comment:
Should it say `...is already share **over** {}`? The share string is the protocol used.

Reply:
Indeed, done!